cortex.tf

```diff
@@ -0,0 +1,144 @@
+locals {
+  cortex = merge(
+    local.helm_defaults,
+    {
+      name                      = "cortex"
+      namespace                 = "cortex"
```

Comment:
maybe default namespace should be the same as prometheus to avoid network policy issue. So maybe `monitoring` ?